packages/doom_emacs.rb

```diff
@@ -0,0 +1,91 @@
+require 'package'
+
+class Doom_emacs < Package
+  description 'An Emacs framework for the stubborn martian hacker'
+  homepage 'https://github.com/hlissner/doom-emacs'
+  version '27.0.91'
+  source_url 'http://git.savannah.gnu.org/cgit/emacs.git/snapshot/emacs-27.0.91.tar.gz'
```

Comment:
Can we use https://git.savannah.gnu.org/cgit/emacs.git/snapshot/emacs-27.0.91.tar.gz?  Since https works.